.github/workflows/linting.yml

```diff
@@ -0,0 +1,27 @@
+name: Linting
```

Comment:
I believe this was copied from another project, but can we please revisit the linting setup before proceeding?  I think there are some issues with it:

- We have "tox -e static" and we have this workflow. It seems inefficient to have both, using similar tools - and I'd prefer the one which I can run locally (tox).  It has happened a few times that I had a change fully passing "tox -e static" locally only to hit problems in this workflow.  It would have been better if all the tools we want to use were included in the tox config so I can use all of them locally.
- I don't think we should use both flake8 and black since both are doing almost the same thing.  Can we just use black?
- isort is good but could we use a pre-commit hook for that instead?  If pre-commit CI is then enabled, that will enforce isort (and the other precommit hooks like black) without needing a separate workflow.
- The line lengths are not configured the same across all tools.  Some places it says 79, another place it says 100.  I don't much care what is the line length for a new repo, but all the tools need to be configured consistently.

So in summary, could we please drop this workflow, drop usage of flake8 altogether, use pre-commit for isort and ensure all tools which need a configured line length are using the same value?

Reply:
Sounds great. Removed flake8 and the linting workflow, added isort check to tox -e static.